manifests/n/Nostalgia-09/AeroChat/1.0.1-rc.3/Nostalgia-09.AeroChat.locale.en-US.yaml

```
@@ -0,0 +1,21 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Nostalgia-09.AeroChat
+PackageVersion: 1.0.1-rc.3
+PackageLocale: en-US
+Publisher: Nostalgia-09

```

```suggestion
Publisher: Madeline, with special thanks to luisl and the Nostalgia '09 team
```
Should match control panel